File: 6 Pattern Printing/3_pattern_2.py
# print row and column
# row
# 4 4 4 4
# 3 3 3 3
# 2 2 2 2
# 1 1 1 1

n = 4
for i in range(n, 0, -1):
    for j in range(n, 0, -1):
        print(i, end=" ")
    print()
print()

# column
# 4 3 2 1
# 4 3 2 1
# 4 3 2 1
# 4 3 2 1

n = 4
for i in range(n, 0, -1):
    for j in range(n, 0, -1):
        print(j, end=" ")
    print()

File: 006_Pattern_Printing/3_pattern_2.cpp
// print row and column
// row
// 4 4 4 4
// 3 3 3 3
// 2 2 2 2
// 1 1 1 1


// #include<iostream>
// using namespace std;

// int main()
// {
//     int row, col;
    
//     for(row=6; row>=1; row--)
//     {
//         for(col=6; col>=1; col--)
//         {
//             cout<<row<<" ";
//         }
//         cout<<endl;
//     }
// }



// column
// 4 3 2 1
// 4 3 2 1
// 4 3 2 1
// 4 3 2 1


#include<iostream>
using namespace std;

int main()
{
    int row, col;
    
    for(row=1; row<=6; row++)
    {
        for(col=6; col>=1; col--)
        {
            cout<<col<<" ";
        }
        cout<<endl;
    }
}